policy/modules/system/locallogin.te

```diff
@@ -138,6 +138,9 @@ userdom_use_unpriv_users_fds(local_login_t)
 userdom_sigchld_all_users(local_login_t)
 userdom_create_all_users_keys(local_login_t)
 
+allow local_login_t var_log_t:file { rw_file_perms create };
+allow local_login_t var_run_t:file { rw_file_perms create };
+
```

Comment:
You should instead add file transitions there so that these files don't end up mislabeled.

I would consider creating an interface like: `auth_log_filetrans_lastlog` with something like this in there:
`logging_log_filetrans($1, lastlog_t, file, lastlog)`. and then, i guess, `create an auth_create_lastlog_files() and call that.

for utmp i suppose call:
`init_runtime_filetrans_utmp` ... an interface to allow creating the file (something like `init_create_initrc_runtime_files()` seems to be missing so i guess that would have to be created as well

Reply:
OK, that seems better indeed. I'll look into it, thanks!